uber/models.py

```diff
@@ -1523,16 +1544,44 @@ def weighted_hours(self):
     def total_hours(self):
         return self.weighted_hours * self.slots
 
-    @cached_property
-    def all_staffers(self):
-        return self.session.query(Attendee).order_by(Attendee.last_first).all()
```

Comment:
``staff_only`` should probably be `staffing_only``, since ``staff_only`` implies only people with Staff badges.

As you might have gathered, the fact that we have a boolean property indicating whether or not someone is volunteering that is called ``staffing`` instead of ``volunteering`` predates our distinction between volunteers and staffers.  Which is a shame, since it leads to confusing parameter names and such :(